src/view/components/customerIdWrapper.jsx

```diff
@@ -0,0 +1,200 @@
+import "regenerator-runtime"; // needed for some of react-spectrum
+import React, { useState, useEffect } from "react";
+import { FieldArray } from "formik";
+import Textfield from "@react/react-spectrum/Textfield";
+import Checkbox from "@react/react-spectrum/Checkbox";
+import Select from "@react/react-spectrum/Select";
+import FieldLabel from "@react/react-spectrum/FieldLabel";
+import Button from "@react/react-spectrum/Button";
+import Well from "@react/react-spectrum/Well";
+import Heading from "@react/react-spectrum/Heading";
+import Delete from "@react/react-spectrum/Icon/Delete";
+import ModalTrigger from "@react/react-spectrum/ModalTrigger";
+import Dialog from "@react/react-spectrum/Dialog";
+import "@react/react-spectrum/Form"; // needed for spectrum form styles
+import PropTypes from "prop-types";
+import WrappedField from "./wrappedField";
+import authenticatedStateOptions from "../constants/authenticatedStateOptions";
+import getDefaultCustomerId from "../utils/getDefaultCustomerId";
+import ErrorMessage from "./errorMessage";
+import InfoTipLayout from "./infoTipLayout";
+
+function CIdWrapper({ values }) {
+  // const [namespaceOptions, setNamespaceOptions] = useState([]);
+  const [showStuff, setShowStuff] = useState({});
```

Comment:
showStuff isn't very descriptive.  I'd suggest naming it "errorMessage". For example:
```
const [identityNamespaces, setIdentityNamespaces] = useState([]);
const [errorMessage, setErrorMessage] = useState("");
if (errorMessage) {
  return ...;
}
if (identityNamespaces.length > 0) {
  return ...;
}
return ...;
```